transport/http/transport.go

```diff
@@ -80,6 +94,14 @@ func Tracer(tracer opentracing.Tracer) TransportOption {
 	}
 }
 
+// Socks5Proxy sets up outbounds connections to run through the specified proxy
+// of the format host:port
+func Socks5Proxy(hostPort string) TransportOption {
```

Comment:
Go standard library establishes a precedent for hostport string of form "host:port", e.g., net.Listen.